src/pytorch_tabular/tabular_model_tuner.py

```diff
@@ -348,4 +371,12 @@ def tune(
         best_params = trials_df.iloc[best_idx].to_dict()
         best_score = best_params.pop(metric_str)
         trials_df.insert(0, "trial_id", trials)
-        return self.OUTPUT(trials_df, best_params, best_score)
```

Comment:
Can we add `best_model` to the namedtuple `OUTPUT`? and edit the documentation accordingly?  I feel its a bit cleaner, and we can add another cell in the tutorial showing the best model is available in the namedtuple and one can access it like this..

Reply:
Added!